client/api/search.ts

```diff
@@ -76,6 +76,22 @@ export function searchRaw<T>(args: ISearchAPIParams): Promise<IRestApiResponse<T
     );
 }
 
+export const searchRawAndStore = <T extends Array<IEventOrPlanningItem>>(args: ISearchAPIParams) => (
+    (dispatch: any): Promise<IRestApiResponse<T>> => superdeskApi.dataApi.queryRawJson<IRestApiResponse<T>>(
```

Comment:
Please don't use redux actions in `client/api` module. We want these to be pure python functions in the way they're executed.

Instead you can obtain the `dispatch` function from the API, i.e.
`const {getState, dispatch} = planningApi.redux.store;`

And modify the import
`import {superdeskApi} from '../superdeskApi';`
to be
`import {superdeskApi, planningApi} from '../superdeskApi';`